src/main/resources/static/css/style.css

```diff
@@ -2983,3 +2983,9 @@ header .logout {
 .text-group .text-only {
   padding-left: 180px;
 }
+.control-group .text {
+  padding-top: 5px;
+}
+.courses-top-control {
+  margin-bottom: 12px;
+}
```

Comment:
Why do you have such many strings in css? Do you just edit bootstrap css file instead of creating new one? If so, it is bad. You do bootstrap not updatable in your project, and your css code not reusble.